### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by SunnyGarg**

### CODE REVIEW

File: ghi.txt.txt [hunk 1]
```diff
@@ -1 +1 @@
-fdddffd
\ No newline at end of file
+fdddffdffgfg
```
Comment (prakasbabu) [inline ghi.txt.txt:1]: some comments to passed

**Review by prakasbabu — CHANGES_REQUESTED** [review]

File: ghi.txt.txt (hunk 1 above)
Comment (SunnyGarg) [reply ghi.txt.txt:1]: I have made changes, please review again.

**Review by prakasbabu — APPROVED** [review]

File: ghi.txt.txt (hunk 1 above)
Comment (mayaban) [inline ghi.txt.txt:1]: what is this code?

Comment (mayaban) [reply ghi.txt.txt:1]: this is stupid guys!!!

**Review by mayaban — APPROVED** [review]
good job guys

**Pull request merged**